src/graphql/schema.graphql

```diff
@@ -0,0 +1,51 @@
+type User {
```

Comment:
How would add comments to document your schema?

Reply:
Comments can be added with triple and single quotes per the [spec](https://spec.graphql.org/October2021/#sec-Descriptions). Added a couple as example.